packages/sdk/src/evm/core/classes/erc-1155.ts

```diff
@@ -272,6 +272,40 @@ export class Erc1155<
     },
   );
 
+  /**
+   * Transfer multiple NFTs
+   * 
+   * @remarks Transfer multiple NFTs from the connected wallet to another wallet.
+   * 
+   * @example
+   * ```javascript
+   * // Address of the wallet you want to send the NFT to
+   * const toAddress = "{{wallet_address}}";
+   * // The token IDs of the NFTs you want to send
+   * const tokenIds = [0, 1, 2];
+   * // How many copies of the NFTs to transfer
+   * const amounts = [1, 2, 3];
+   * await contract.erc1155.transferBatch(toAddress, tokenIds, amounts);
+   * ```
+   * 
+   * @twfeature ERC1155BatchTransferable
+   */
+  transferBatch = /* @__PURE__ */ buildTransactionFunction(
+    async (
+      to: AddressOrEns,
+      tokenIds: BigNumberish[],
+      amounts: BigNumberish[],
+      data: BytesLike = [0],
```

Comment:
can you add an optional `from` here? if not passed in, defaults to current signer

Reply:
we don't do that for `transfer`, should I add it there too

Reply:
added [here](https://github.com/thirdweb-dev/js/pull/2220/commits/80c5e080efe8c8092c02a42be730232e2129efe1)